bin/setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Where is this file sourced? Do we really need it? If the variables are needed, nextflow's [env scope](https://www.nextflow.io/docs/latest/config.html#scope-env) should be considered

Reply:
I don't really understand the question...
The code was taken from the biscuit-scripts section, in [here](https://github.com/huishenlab/biscuit/tree/master/scripts)
This file is part of the biscuit_QC process, and needed for the QC.sh file. The variable are set through the QC.sh script, so I don't really understand the second part...

Reply:
I meant that I can't find the place where `setup.sh` is executed. All it seems to do is setting some environment variables, which are not even used in `QC.sh`

Reply:
They are used and referred to in QC.sh.
Can one unit the files? maybe yes, it's only setting of variables for the QC.sh

Reply:
Looking at https://github.com/huishenlab/biscuit/blob/master/scripts/QC.sh
I don't see a reference to any of these variables

Reply:
Yes. The biscuit team told me it's "a new QC script which doesn't need the setup script. You are welcome to use it, but it is incompatible with the current version of MultiQC", so until they update the MultiQC, they suggested me to keep using my own script, which is the older verstion.
Anyway, I'm working on adding it to the recipe, so it will be update automatically... No?

Reply:
It would be even better to add support to MultiQC instead of writing your own script, if possible. Then everyone benefits 😉 

I'm doing a period of work on MultiQC at the moment and reviewing pull requests, will hopefully be getting a new release out relatively soon.

Reply:
It's not my own script. It's the biscuit QC.sh old script, which now has support in the MultiQC.
The biscuit team changed the QC.sh script, so the newer script, and they need to update also the MultiQC and they didn't do it yet.